### PR TITLE
Gives medical a prepared belt.

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
The first aid kit has been removed from the doctors hands, and has been replaced with a medical belt variant that comes with a brute and burn automender, 2 empty hyposprays, an upgraded health analyzer, and a defib. Also gives this belt to medical specialist. 

## Why's this needed? 
QoL and it should make it much more clear to new players what gear theyre expected to use. 

## Changelog

```changelog
(u)Ikea
(*)Medical doctors, directors and specialists now come with a belt filled with automenders, one empty hyposprays, a health analyzer and a defib.
```

### CODE REVIEW

**Comment by DisturbHerb** [comment]
im having a hard time trying to figure out if this is a good idea, does anyone concur or am i alone

**Comment by ThatFiveGuys** [comment]
> im having a hard time trying to figure out if this is a good idea, does anyone concur or am i alone

What do you see as the downside? To me the majority of stuff in the first aid kit is useless and I dont like the doctors huddling around a medical fabricator to get the things they actually need for there job

**Comment by Debi-sys** [comment]
> im having a hard time trying to figure out if this is a good idea, does anyone concur or am i alone

I feel like its an amazing idea, i feel like most people put the inside of the kid into the belt anyways.

**Comment by t-toasted** [comment]
3 hypos is a lot, especially on maps that have some spawn by default. At most, you'll see a saline/charcoal/epi mix. Also, on RP, many people do use their patches and the epi autoinjector. Hell, I love slapping the patches on people. The pill is very useful in a pinch. And the autoinjector is a good one time use to have.

**Comment by ThatFiveGuys** [comment]
Id rather have an excess of hyposprays in belts then a shortage. I think its much more interesting for new players to go "I have three hyposprays, one needs to be epipherine, the other needs to be charcoal.. hmm what should I put in the third?" Then "Okay I have two hyposprays, I need epipherine in the other, and charcoal in the other". 

While the patches and pills can be cute for roleplay, i dont think its fair to communicate to new players that these are the “proper” things to use considering theyre worse. Not to mention these things are pretty useless on classic. If players wish to grab these things they can head to a first aid locker/nanomed and grab them with minimal issue.

**Comment by wolfolotl** [comment]
dislike how many hypos there are especially since they dump out certain medchems automatically, would prefer a syringe and like a starting bottle of saline glucose or epi or something

**Comment by ThatFiveGuys** [comment]
> dislike how many hypos there are especially since they dump out certain medchems automatically

This seems much better addressed by just having them refuse to add these chemicals or something like that, and seems like an unrelated issue.

> plus I feel like it takes away from a healthy learning environment to have the basic equipment equal The Most Streamlined Stuff. not great for all the weird edge cases that come with medical that new doctors will inevitably encounter somewhere along the line

Right now instead of “new doctors occasionally are confused for edge cases” its “new doctors dont have any clue how to do effective doctoring and play very poorly” and i think the former is a much better issue to have then the latter.

**Comment by BatElite** [comment]
At rate, should we have a forum topic about this or what?

**Comment by ThatFiveGuys** [comment]
> At rate, should we have a forum topic about this or what?

I wouldnt be opposed however i dont see much of a point. The majority of the criticism is due to this pr inconvinencing a specific roleplay server experienced doctor playstyle whose playstyle boils down to slowing down doctoring to take more time with patients. While its a perfectly fine playstyle, this pr’s purpose is primarily to communicate more clearly to classic new players who dont have time to fiddle around with entirely syringes and patches what gear youre generally going to use, and for the qol of most players, not the qol of players who prefer a slower playstyle.

**Comment by Bilo216** [comment]
I had to think a bit about this one, so this might be a bit long. I see where this comes from, as it is a common loadout for doctors even on RP. However, I still believe that 3 hyposprays is a Lot. I understand your point of wanting to guide new doctors to more efficient loadouts but also… why to that extreme? Any doctor will recommend the use of hypos to other doctors. You can print them very easily, so there cannot really be any shortage of them. If the objective is to show new doctors what a hypo is and how to use it, one single hypo can do the job, and from there they have room to mess around and try their belt loadouts. Having no idea what you are doing and learning from it is part of the game, and I don't see how having three hypos would help with that much, really. Learning what patches and autoinjectors are, as much “worse” they are to menders and hypos, is still a good step to take. I do see this PR with an objective more leading into a “saving time at the beginning of the round” rather than something to help guide new doctors. I agree that this should be taken to the forums to see what more people think about it.

**Comment by ThatFiveGuys** [comment]
>However, I still believe that 3 hyposprays is a Lot. I understand your point of wanting to guide new doctors to more efficient loadouts but also… why to that extreme?

I really dont get why everyone is pretending like 3 is an extreme amount? Its the needed amount if you want to carry a decent amount of chems.

>I understand your point of wanting to guide new doctors to more efficient loadouts but also… why to that extreme? Any doctor will recommend the use of hypos to other doctors. You can print them very easily, so there cannot really be any shortage of them. 

Someone there to teach oftentimes isnt a thing and it's always good for the game to guide towards being good. If hyposprays arent worth nothing, that also means there cant be a surplus because having more doesnt really matter and they can just be thrown away without a cost. 

> If the objective is to show new doctors what a hypo is and how to use it, one single hypo can do the job, and from there they have room to mess around and try their belt loadouts. Having no idea what you are doing and learning from it is part of the game, and I don't see how having three hypos would help with that much, really. 

Doctors already get a lot of stuff that is and that can be overwhelming. I think, hey these are your automenders explicitly for these damages, here are your empty hyposprays with inject on em (should probably look into what chems work for these), and here is a health analyzer, is much more clear then "Hey heres a patch, syringe, hypospray, pill, also half of these things youre generally not going to use but we're not going to tell you which because theyre in the same place good luck."

**Comment by NightmareChamillian** [comment]
Instead of 3 hyposprays, do you think you can move the defibrilator and crowbar from the medical doctor's starting backpack into the belt? 3 roundstart hyposprays is way more than I ever see myself using, and oversaturates the round with them. (Especially since pre-change MDs start with 0.)

I'm all for moving away from MDs starting with a first aid kit inhand, but if we're doing that, we may as well carryover what they already start with instead of add new things.

**Comment by ZeWaka** [comment]
i think one of the hyposprays should be a syringe

**Comment by ThatFiveGuys** [comment]
I swapped a hypospray with a syringe, and reintroduced doctor_spawn into code as a "prepared" first aid kit, with menders, an upgraded analyzer, a hypospray, and small beakers of epi, salbutamol and charcoal.

**Comment by ThatFiveGuys** [comment]
Swapped syringe with the defib and removed the defib in backpacks

**Comment by Conchuckter** [comment]
Even two hyposprays is a bit much, replacing one with a syringe would be a good idea. A little preparation at the start of the game is good for gameplay and means doctors have to interact with other departments to get mats, etc.

**Comment by ThatFiveGuys** [comment]
Honestly the only reason why people use so few hyposprays right now is because no one wants to wait 45 seconds. Standing in a queue to wait on other people is not fun and i wont complain if this kills it.

**Comment by Josephin23** [comment]
> Even two hyposprays is a bit much, replacing one with a syringe would be a good idea. A little preparation at the start of the game is good for gameplay and means doctors have to interact with other departments to get mats, etc.

Hyposprays are incredibly cheap material-wise, they're only tedious to make because they take 13 seconds. They're a basic item for doctors, they should be accessible,

File: code/obj/item/storage/med_chem.dm [hunk 1]
```diff
@@ -30,14 +30,14 @@
...
-		/obj/item/device/analyzer/healthanalyzer/upgraded,\
-		/obj/item/reagent_containers/emergency_injector/epinephrine)
+	prepared
+		spawn_contents = list(/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/device/analyzer/healthanalyzer/upgraded,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/glass/beaker/antitox,
+		/obj/item/reagent_containers/glass/beaker/epinephrine,
+		/obj/item/reagent_containers/glass/beaker/salbutamol)
```
Comment (Tarmunora) [inline code/obj/item/storage/med_chem.dm:40]: hypos and beakers do not belong in first aid kits.

File: code/obj/item/storage/backpack_belt_etc.dm [hunk 2]
```diff
@@ -638,6 +638,13 @@
 	can_hold = list(/obj/item/robodefibrillator)
 	in_list_or_max = 1
 
+/obj/item/storage/belt/medical/prepared
+	spawn_contents = list(/obj/item/reagent_containers/mender/brute,
+	/obj/item/reagent_containers/mender/burn,
+	/obj/item/reagent_containers/hypospray = 2,
```
Comment (Tarmunora) [inline code/obj/item/storage/backpack_belt_etc.dm:644]: ditch at least one of the hypos from here.

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/item/storage/med_chem.dm (hunk 1 above)
Comment (ThatFiveGuys) [reply code/obj/item/storage/med_chem.dm]: Swapped with large auto injectors (know its a nukie thing however the aid kit will also be an admin thing so, tell me if bad)

File: code/obj/item/storage/backpack_belt_etc.dm (hunk 2 above)
Comment (ThatFiveGuys) [reply code/obj/item/storage/backpack_belt_etc.dm]: Removed one

**Pull request merged**